R/serialized_tests.R

```diff
@@ -209,3 +209,10 @@ style_op <- function(text) {
   transformed_text
 
 }
+
+#' Create the path to a test that file
+#' @param ... Arguments passed to [file.path()] to construct the path after
+#'   ".../tests/testthat/"
+testthat_file <- function(...) {
```

Comment:
This might be better suited in a `tests/testthat/helper-*.R` file.

Reply:
1. ups, forgot to knit `.Rmd` to `md`.
2. Ok, but then we won't get `roxygen2` documentation...

Reply:
Yes, I forgot. Let's keep it here.